four_wheel_steering_controller/include/four_wheel_steering_controller/speed_limiter.h

```diff
@@ -0,0 +1,137 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2013, PAL Robotics, S.L.
```

Comment:
I agree with you but is it fine to depend on the diff_drive_controller ? Or is it better to create another package with speed_limiter and odometry library ?